mlrun/api/api/endpoints/projects.py

```diff
@@ -323,6 +325,89 @@ async def get_project_summary(
     return project_summary
 
 
+@router.post(
+    "/projects/{name}/load", response_model=mlrun.common.schemas.BackgroundTask
+)
+async def load_project(
+    name: str,
+    url: str,
```

Comment:
This is a good point, I don't really thought about it.

Since this function is using the `mlrun.load_project` function, we are limited to this function abilities.
I see there is a `secrets` parameter, with this description (from docstring):
key:secret dict or SecretsStore used to download sources

I think we should talk about this offline.